.github/workflows/build.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build for slsa
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v*'
+      
+jobs:
+  build:
+    runs-on: "ubuntu-latest"
+    outputs:
+      hashes: ${{ steps.hash.outputs.hashes }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+      - name: Generate checksum
+        id: hash
+        run: |
+          # Install 'build' and create sdist & wheel
+          python -m pip install build
+          python -m ensurepip --upgrade
+          python -m pip install -r requirements.txt
+          python -m build
+
+          # Gather hashes of all files
+          set -euo pipefail
```

Comment:
Not sure why we would need this here, if a command fails, the step will fail too